spec/workers/csv_export_worker_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'govuk_sidekiq/testing'
+
+RSpec.describe CsvExportWorker do
+  let(:search_params) do
+    {
+      date_range: 'past-30-days',
+      organisation_id: 'org-id',
+      document_type: 'news_story'
+    }
+  end
+
+  let(:document_types) do
+    [{ id: 'case_study', name: 'Case study' },
+     { id: 'guide', name: 'Guide' },
+     { id: 'news_story', name: 'News story' },
+     { id: 'html_publication', name: 'HTML publication' }]
+  end
+
+  let(:organisations) do
+    [{ name: 'org', id: 'org-id', acronym: 'OI', },
+     { name: 'another org', id: 'another-org-id', },
+     { name: 'Users Org', id: 'users-org-id', acronym: 'UOI', }]
+  end
+
+  let(:content_items) do
+    [
+      {
```

Comment:
We are setting many values in the test that are not in used. Factories might help here. Have you considered using FactoryBot to simplify these tests? This could also benefit the feature test.

Reply:
Agreed - however, I suggest this being a separate PR as we have a few other test cases where the FactoryBot could be used. It would take a bit effort to refactor this.